src/components/ReadBookPage.tsx

```diff
@@ -117,6 +117,12 @@ export const ReadBookPage: React.FunctionComponent<{
     useSetBrowserTabTitle("Play"); // Note that the title comes from the ?title parameter, if present. This "Play" will not normally be used.
     const handleMessageFromBloomPlayer = useCallback(
         (event: MessageEvent) => {
+            if (event.data && event.data.event) {
+                const eventName = event.data.event;
+                if (eventName === "addBloomLibraryLocationToUrl") {
+                    return; // intercepted a message for 'receiveBloomLibraryMessage(e)'
+                }
+            }

```

Comment:
Seems like we would benefit from making this more robust.
Should EmbeddingHost.tsx export a list of events which this should ignore? Better ideas?
Could use expanded comments, too.